collections/_infographics/emissions/emission-reduction-potential.md

```diff
@@ -3,12 +3,139 @@ layout:        infographic
+
+{% include preview-box.html
+    title="Comparison of energy scenarios"
+    text="The estimates of emissions in the power sector are based on our comparison of different scenarios for the development of the Czech electricity sector until 2030."
+    slug="transition-scenarios-comparison-czechia"
+%}
+
+The Czech Republic's [draft](/studies/2019-scenario-necp) National Energy and Climate Plan (**NECP**) of February 2019 expects a gradual increase in the share of renewable energy sources and a related reduction in the emission footprint of the Czech electricity sector. According to this scenario, 1.8 GW of photovoltaic power plants (about +85% compared to the current situation) and 0.7 GW of wind power plants should be installed. If this plan is implemented, **the resulting emissions from electricity generation should fall by around 5.9 million tonnes of CO<sub>2</sub>eq by 2030**.
+
+Higher savings would be achieved by implementing a [scenario](/studies/2018-scenario-energynautics) developed in 2018 by the German consultancy **Energynautics** on behalf of Czech environmental NGOs. According to this study, which primarily focused on the electricity grid's stability when most coal-fired power plants are shut down, **the resulting emissions would fall by around 16 million tonnes of CO<sub>2</sub>eq**. The study envisaged replacing the capacity of the decommissioned coal-fired power plants mainly with new photovoltaic (+162 % of installed capacity) and wind (+635 % of installed capacity, equivalent to just under six hundred newly installed wind power plants with a unit capacity of 3 MW). In this case, grid stability would also be supported by new gas-fired power plants (+95 % of installed capacity).
+
+According to the **McKinsey** study, implementing the decarbonisation <!--[-->scenario<!--](/studie/2020-scenar-mckinsey)--> would achieve similar emission savings in the electricity industry. This study models the development in the energy sector, as well as in industry, agriculture, transport and other sectors, looking for the cost-optimal way to decarbonise. In addition to emissions reductions by 2030, it also examines full climate neutrality by 2050. Savings in 2030 are achieved by a similar reduction in coal-fired power generation as in the Energynautics study. Still, compared to that study, McKinsey assumes a more gradual development of solar (+120% of installed capacity) and wind (+230% of installed capacity). The difference in the electricity balance is then offset by imports from abroad.
```

Comment:
Unable to find this study in English. This is the Czech link: https://faktaoklimatu.cz/studie/2020-scenar-mckinsey